public/app/features/plugins/plugin_loader.ts

```diff
@@ -200,11 +217,22 @@ export async function importPluginModule(path: string, version?: string): Promis
       return builtIn;
     }
   }
+
+  // This code cannot work in a nodejs environment and requires a real browser for the sandboxing mechanism to function.
+  if (!isAngular && config.featureToggles.pluginsFrontendSandbox && process.env.NODE_ENV !== 'test') {
```

Comment:
Maybe move this into a function called something like: `isFrontendSandboxSupported`.

```suggestion
  if (isFrontendSandboxSupported(isAngular)) {
```

Reply:
```ts
function isFrontendSandboxSupported(isAngular: boolean): boolean {
  // This code cannot work in a nodejs environment and requires a real browser for the sandboxing mechanism to function.
return !isAngular && config.featureToggles.pluginsFrontendSandbox && process.env.NODE_ENV !== 'test';
}
```

Reply:
Later on there will be more configuration-based  conditions. I move the code to a function as suggested for now but this will evolve into something more complex.